doc/panama_ffi.md

```diff
@@ -72,7 +128,9 @@ At the core of Panama foreign function support we find the `CLinker` abstraction
 
 ```java
 interface CLinker {
+    MethodHandle downcallHandle(MethodType type, FunctionDescriptor function);
     MethodHandle downcallHandle(Addressable func, MethodType type, FunctionDescriptor function);
+    MemorySegment upcallStub(MethodHandle target, FunctionDescriptor function, ResourceScope scope);
     MemorySegment upcallStub(MethodHandle target, FunctionDescriptor function);
     ...
```

Comment:
yeah - since the examples never mention it, I didn't want to dive too much into that. I think I will simplify this further and just keep two methods (but then say that there are other overloads available).

Reply:
Ok, sounds good